### DISCUSSION TITLE
channel to chat and network on IRC

### DISCUSSION THREAD

**Discussion by vlcg-gif** [hylo-lang/hylo#1343] [Show and tell]
I created a chat channel on IRC, on the Libera.chat server, to join it is very easy, choose a client on IRC and access /join #hylo-lang or access https://web.libera.chat/#hylo-lang Thank you and let's meet there!!!

**Comment by dabrahams** [comment]
As we already have a slack channel, why would we fragment interactive discussion by doing some of that on IRC?
